src/sync.ts

```diff
@@ -100,13 +100,11 @@ const MSC2716_ROOM_VERSIONS = [
 function getFilterName(userId: string, suffix?: string): string {
     // scope this on the user ID because people may login on many accounts
     // and they all need to be stored!
-    return "FILTER_SYNC_" + userId + (suffix ? "_" + suffix : "");
+    return `FILTER_SYNC_${userId}${suffix ? "_" + suffix : ""}`;
 }
 
 function debuglog(...params) {
-    if (!DEBUG) {
-        return;
-    }
+    if (!DEBUG) return;
```

Comment:
Shouldn't it be kept?

https://github.com/matrix-org/eslint-plugin-matrix-org/blob/4dd27e4b9d4313cf48fa02125a984e31ee7f6ce8/javascript.js#L20

Reply:
That rule says it only is needed on multi-line things, 1 != multi